config/crd/bases/osp-director.openstack.org_openstackdeploys.yaml

```diff
@@ -72,6 +72,9 @@ spec:
                 description: ConfigVersion the config version/git hash of the playbooks
                   to deploy.
                 type: string
+              deployIndentifer:
```

Comment:
Does it make sense to expose this to the user vs always generating a random uuid? If they did want to set this how would they determine the existing value?